roles/ssh_keys/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+ssh_key_name_src: "id_rsa_vagrant"
```

Comment:
The assumption being that you'd have a separate SSH key for vagrant?

Reply:
Yeah, I didn't know if people would want their actual key copied by default, but you can set this parameter in the boxes.yml.

Reply:
Another approach is to suggest SSH agent forwarding. I recall you can use ```vagrant ssh box -- -A``` or add ```config.ssh.forward_agent``` to the ```Vagrantfile```.

Reply:
Wow, agent forwarding is awesome! *But*, I don't think it solves the known_hosts issue, unless you have more information?

Reply:
No, but known_hosts copying is much less of an issue security wise so we could do that by default or just ensure github.com is in the system known hosts.